### ISSUE TITLE
airmail beta

### ISSUE THREAD

**Issue by chrisRidgers** [caskroom/homebrew-versions#983]
As it currently stands (I think) airmail-beta.rb is out of date, and the in app upgrade process is a manual download/ install procedure.

Anyone able to maintain it so it stays up to date?

**Comment by vitorgalvao** [comment]
Unlikely. We don’t have maintainers for specific casks (though some users do tend to keep certain casks they’re interested in, up to date).

You can always submit a PR to update the latest version, though.

**Comment by chrisRidgers** [comment]
I took a look at the file:

https://github.com/caskroom/homebrew-versions/blob/master/Casks/airmail-beta.rb

But wasn't sure on the origin of the file, I've not seen a source link like that one before.  I thought I'd ask in case someones hosting it on a private server someone/ is responsible for keeping it up to date.

Although I've just noticed that it is the actual url for the beta download.

If wanna go about submitting a PR, is there a tried and tested way for testing for a new version of the file?  I'd rather not do manual update so the next time they update the file it breaks again.
